Makefile

```diff
@@ -0,0 +1,41 @@
+PACKAGE_NAME=riemann_client
```

Comment:
Testing and build steps should use `tox` and go in `tox.ini` - this shouldn't be replicated in a Makefile.